assignment1.txt

```diff
@@ -0,0 +1,21 @@
+Priyanka Musti
+
+1.
+
+A compiler is something that translates the code into something that the computer understands and can carry out.
```

Comment:
right, more specifically, a compiler takes your code written in C, removes the whitespace, places this into a parse tree, optimizes it, and then takes that parse tree and translates it into assembly code.